File: Heat/ViewModels/Invoices/confirmedInvoicesGridViewModel.vb
Imports System.ComponentModel.DataAnnotations

Namespace ViewModels.Invoices
    ''' <summary>
    ''' Un modello per la visualizzazione della griglia delle fatture confermate. 
    ''' </summary>
    ''' <remarks></remarks>
    Public Class confirmedInvoicesGridViewModel
        Property ID As Integer

        <Display(name:="Cliente")> _
        Property Customer As String

        <Display(name:="Data emissione")> _
        <DisplayFormat(dataformatstring:="{0:d}")> _
        Property InvoiceDate As DateTime

        <Display(name:="Numero")> _
        Property InvoiceNumber As String

        <Display(name:="TOTALE")> _
        Property TotalAmount As Decimal

        <Display(name:="IVA")> _
        Property TaxesAmount As Decimal

        <Display(name:="Imponibile")> _
        Property TaxableAmount As Decimal

        <Display(name:="Pagamento")> _
        Property Payment As String


    End Class
End Namespace

File: Heat.ConvertedToC#/ViewModels/Invoices/confirmedInvoicesGridViewModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Heat.ViewModels.Invoices
{
    /// <summary>
    /// Un modello per la visualizzazione della griglia delle fatture confermate. 
    /// </summary>
    /// <remarks></remarks>
    public class confirmedInvoicesGridViewModel
	{
		public int ID { get; set; }

		[Display(Name = "Cliente")]
		public string Customer { get; set; }

		[Display(Name = "Data emissione")]
		[DisplayFormat(DataFormatString = "{0:d}")]
		public DateTime InvoiceDate { get; set; }

		[Display(Name = "Numero")]
		public string InvoiceNumber { get; set; }

		[Display(Name = "TOTALE")]
		public decimal TotalAmount { get; set; }

		[Display(Name = "IVA")]
		public decimal TaxesAmount { get; set; }

		[Display(Name = "Imponibile")]
		public decimal TaxableAmount { get; set; }

		[Display(Name = "Pagamento")]
		public string Payment { get; set; }


	}
}

